tests/test_group.py

```diff
@@ -2,9 +2,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import time
 from random import randrange
-
+import uuid
```

Comment:
@davehunt I will move the library one line up. Regarding running the test locally, I couldn't as need to set up my credentials. Sorry about that, but I am pretty sure it will run neatly. I will create the accounts soon.